desktop/CMakeLists.txt

```diff
@@ -16,6 +16,15 @@ set(CMAKE_OSX_DEPLOYMENT_TARGET "10.9")
 message(STATUS "EXTERNAL_MODULES_DIR: ${EXTERNAL_MODULES_DIR}")
 
 string(REGEX MATCH "BUILD_FOR_BUNDLE" BUILD_FOR_BUNDLE "${CMAKE_CXX_FLAGS}")
+
+# It is important to distinguish between directory hierarchies
+# in release and dev builds. Release contains .env
+# in the root dir of the build, while dev build's current dir
+# is status-react/node_modules/react-native, hence the need
```

Comment:
Relative path depends on current inner implementation in react-native-config module ( https://github.com/status-im/react-native-config/blob/master/desktop/reactnativeconfig.cpp.in#L52 )
Since, currently, it resolves full path to the executable and then apply relative path to config file, for dev builds, it's required to jump 2 folders up from `/desktop/bin` directory with `StatusIm` executable.
We might want to change current path resolution inside react-native-config to something more flexible / easier to understand.

Reply:
thank you @MaxRis for correcting! Fixed